src/backend/aspen/covidhub_import/implementation.py

```diff
@@ -103,8 +120,23 @@ def import_project(
         group: Group = session.query(Group).filter(Group.id == aspen_group_id).one()
 
         logger.info("Retrieving from COVIDHUB DB...")
-        consensus_genomes: Collection[ConsensusGenome] = (
-            covidhub_session.query(ConsensusGenome)
+        group_czbids: Collection[CZBID] = (
+            covidhub_session.query(CZBID)
+            .join(Project)
+            .filter(Project.rr_project_id == rr_project_id)
+            .options(
+                selectin_polymorphic(CZBID, [DphCZBID, InternalCZBID]),
+                joinedload(CZBID.genome_submission_info),
+            )
+            .all()
+        )
+
+        czbid_lookup_by_str: Mapping[str, CZBID] = {
+            czbid.czb_id: czbid for czbid in group_czbids
```

Comment:
Why do we do this str comparison?

Reply:
There is no string comparison...?
This is to look up a czbid based on the string.